main/document/document.php

```diff
@@ -744,8 +744,8 @@
 
         // Remove media=screen to be available when printing a document
         $new_content_html = str_replace(
```

Comment:
What will happen with old files that still have this editor.css?
Old documents will work normally? 
Better to leave it like that ...

Reply:
Here the certificates will be displayed normally.